uint16 id
uint16 numOfPatches
geometry_msgs/Vector3 normal
geometry_msgs/Point centroid
geometry_msgs/Point[] vertices
